src/selectors/profile.js

```diff
@@ -199,6 +199,21 @@ export const getGlobalTrackReferences: Selector<
     }))
 );
 
+export const getHasPreferenceMarkers: Selector<boolean> = createSelector(
+  getThreads,
+  threads => {
+    return threads.some(({ stringTable, markers }) => {
+      /*
+       * Does this particular thread have a Preference in it?
+       */
+      const indexForPreferenceString = stringTable.indexForString(
+        'PreferenceRead'
+      );
+      return markers.name.some(name => name === indexForPreferenceString);
```

Comment:
See the other comment below too: It's not clear to me what `name` and `type` should be, and if we can have several `name`s in the future, but only one type, maybe it's better to use the type here.
What do you think?